package.json

```diff
@@ -27,7 +27,7 @@
     "dist:win64": "build --windows --x64"
   },
   "dependencies": {
-    "electron-updater": "^2.17.6",
```

Comment:
Wait, this is still using the caret, so on NPM 3, `electron-updater@2.17.8` is used.
Should we just un-caret everything? Should we assume nobody is using `npm@3`?

I don't think that there's really that much of a downside to specifying specific versions in our `package.json`

Reply:
Gah, that was unintentional again. 🤦‍♂️